app/controllers/sites/new.js

```diff
@@ -126,18 +123,21 @@ export default class CreateSitesNewController extends Controller {
     if (Object.keys(validationResult.errors).length > 0) {
       // Validation failed. Return
       this.validationErrors = validationResult.errors;
+      this.saveButtonPressed = 0;
+      this.hasError = true;
       return;
     }
-
     if (Object.keys(validationResult.warnings).length > 0) {
       this.saveButtonPressed = this.saveButtonPressed + 1;
+      this.hasError = false;
+      this.hasWarning = true;
+      console.log('Dit is je warning', this.hasWarning);
```

Comment:
Remove console log. Zet deze lijn: `this.validationWarnings = validationResult.warnings;` in de plek